bazel/envoy_internal.bzl

```diff
@@ -14,7 +14,7 @@ def envoy_copts(repository, test = False):
         "-Wformat",
         "-Wformat-security",
         "-Wvla",
-        "-std=c++14",
+        "-std=c++17",
```

Comment:
I think we can just remove this line if `.bazelrc` specify it.

Reply:
.bazelrc is specifying cxxopt's and not copt's?

Reply:
This is got rid of. But -std=c++17 is something not applicable to C, so it is in cxxopt rather than copt